src/main/java/fr/esgi/grp9/uparserbackend/user/web/UserController.java

```diff
@@ -4,9 +4,7 @@
 import fr.esgi.grp9.uparserbackend.user.domain.UserServiceImpl;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Revert back old import and verify it works.

Reply:
Done